src/mpm/project.py

```diff
@@ -80,6 +80,7 @@ def _post_load(project):
                 path=project.paths.can,
                 root_type=mpm.canmodel.Root,
                 columns=mpm.canmodel.columns,
+                drop_sources=(models.parameters,),
```

Comment:
This fixes that in CAN tab the enum names are initially displayed correctly. Before CAN table enums always had uuid and displayed enum name correctly only after changing the enum